### DISCUSSION TITLE
How does moving and rotation works for selection?

### DISCUSSION THREAD

**Discussion by Luciogi** [LinwoodDev/Butterfly#553] [Q&A] [answered]
From my usage It is very random , something when I want to move , it rotates, and vice versa

What is does Blue and Red color shows when press Center button of selection?

**Comment by CodeDoctorDE** [comment, marked as answer]
The middle circle is for rotation, when clicking again on the selection you can scale with the same aspect ratio

**Comment by Luciogi** [comment]
OK Blue Color is for scaling with locked aspect ratio
Red is for scaling without locking aspect ratio
